src/rt/sched/multimessage.h

```diff
@@ -69,39 +83,66 @@ namespace verona::rt
       }
     };
 
+    struct Delivered
+    {
+      Body& body;
```

Comment:
Can we have a comment here to document that this is an owning reference. Should the members be potentially const (the closest we can get to public read private write)?